mlflow/utils/file_utils.py

```diff
@@ -355,7 +355,7 @@ def path_to_local_file_uri(path):
     Convert local filesystem path to local file uri.
     """
     path = pathname2url(path)
-    if path == posixpath.abspath(path):
+    if posixpath.isabs(path):
```

Comment:
Fixes an edge case I ran across while debugging where a path is not recognized as absolute if it ends in a slash.